Docs/reference/content/what_is_new.md

```diff
@@ -33,8 +33,7 @@ Other new additions and updates in this beta include:
 * Support for Snappy compression on .NET Core on Windows (in addition
   to existing support on .NET Framework)
 * Support for Zstandard compression on Windows on 64-bit platforms
-* A new default of enabling certificate revocation checking.
-* A new URI option `tlsDisableCertificateRevocationCheck` to disable
+* A new URI option `tlsDisableCertificateRevocationCheck` to enable or disable
```

Comment:
Do we document anywhere what the default value is?

Reply:
I don't know... I was just tweaking what I found already in place. Do we want to investigate more deeply?

Reply:
I'm afraid not explicitly. There's `CheckCertificateRevokation` default:
https://github.com/mongodb/mongo-csharp-driver/blob/master/Docs/reference/content/reference/driver/ssl.md#certificate-revocation-checking
And for `tlsDisableCertificateRevocationCheck` we say that they are connected:
https://github.com/mongodb/mongo-csharp-driver/blob/master/Docs/reference/content/reference/driver/ssl.md#tlsdisablecertificaterevocationcheck

Reply:
Since we mention that ([link](https://github.com/mongodb/mongo-csharp-driver/blob/master/Docs/reference/content/reference/driver/ssl.md#certificate-revocation-checking))
> The .NET Driver now disables certificate revocation checking by default

I am fine with how it is currently described.